### DISCUSSION TITLE
NPM page has (very) bad links

### DISCUSSION THREAD

**Discussion by outlawpractice** [udecode/zustand-x#82] [General]
Link to docs on the NPM page is https://zustand-x.udecode.io, not https://zustand-x.udecode.dev.  The .io link takes you to a site with nastly malware.

Link to the github repo points to https://github.com/udecode/plate, not https://github.com/udecode/zustand-x.

Just thought you should know!

**Comment by zbeyens** [comment]
Thanks for reporting it, it should be fixed in next patch.
